app/components/tags_input/tags_input.css

```
@@ -28,3 +26,27 @@
 .TagsInput tag {
   margin-bottom: 0;
 }
+
+.TagsInput-dropdownItem {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+}
+
+.tagify__dropdown__item--active,
+.tagify__dropdown__item:hover {
+  background-color: var(--color-gray-light);
+}
+
+.TagsInput-name {
+  padding: 0 var(--spacing-unit-xs);
+  border-radius: var(--border-radius);
+  background-color: var(--tag-bg) !important;
```

```suggestion
  background-color: var(--tag-bg);
```
this isn't necessary, right?
I tested it without and it seems fine...